addons/portal_rating/models/rating_rating.py

```diff
@@ -54,3 +55,15 @@ def _synchronize_publisher_values(self, values):
             if not values.get('publisher_id'):
                 values['publisher_id'] = self.env.user.partner_id.id
         return values
+
+    def _to_store_defaults(self):
+        return super()._to_store_defaults() + (
+            [
+                Store.One("message_id", []),
+                Store.One("publisher_id", ["name", "is_company", "user", "write_date"]),
```

Comment:
If you use `avatarUrl` in JS you need `avatar_128` here (and `write_date` doesn't need to be explicit anymore).
I'm not sure what is the purpose of `user` and `is_company`, they don't seem used.